app/controllers/posts_controller.rb

```diff
@@ -154,6 +154,14 @@ def events
     render json: events_json
   end
 
+  def create_doc
+    render :text => "Doc created"
```

Comment:
Please check the email by priyanka titled - 'Appointment json' which says what response is generated by server, on which if the response shows success then you should just call the flash message which says that booking is done